lib/sealField.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const iron = require('iron');
+const nodeify = require('./nodeify');
+
+/**
+ * Seal a field
+ * @method sealField
+ * @param  {Object}         object              The object to seal
```

Comment:
It's not necessarily a model, though I don't know where else it'd be used. Changed.